pages/agent/v3/securing.md.erb

```diff
@@ -102,6 +106,14 @@ do
 done
 ```
 
+## Signing pipelines  
+
+<section class="Docs__troubleshooting-note">
+  <p>Please remember that for maximum security it’s not recommended to use plugins.</p>
+</section>
+
+Suppose using plugins is crucial to your workflow, but you would still like to preserve strong security guarantees. In that case, you can use a third-party tool [buildkite-signed-pipeline](https://github.com/seek-oss/buildkite-signed-pipeline). It adds extra security guarantees around Buildkite’s jobs by allowing the uploaded steps to be signed with a secret shared by all agents. As a result, plugins can run without concerns of tampering by third parties. 
```

Comment:
```suggestion
If using plugins is crucial to your workflow, but you would still like to preserve strong security guarantees, you can use the [buildkite-signed-pipeline](https://github.com/seek-oss/buildkite-signed-pipeline) plugin. It adds extra security guarantees around Buildkite’s jobs by allowing the uploaded steps to be signed with a secret shared by all agents. As a result, plugins can run without concerns of tampering by third parties. 
```

I'm not sure about "Buildkite’s jobs" here.

Reply:
Do you have any suggestions for something more specific? What are we signing? The commands in the plugin, so you that you know that they've not been tampered with?

Reply:
As for "Buildkite's jobs", it's from the description of the tool, verbatim (https://github.com/seek-oss/buildkite-signed-pipeline).
I'll expand about the way the tool works - I originally wanted to keep it _super_ short here.